lib/model/store.dart

```diff
@@ -263,6 +269,18 @@ abstract class GlobalStore extends ChangeNotifier {
 
   Account? getAccount(int id) => _accounts[id];
 
+  Account? get lastVisitedAccount {
+    final id = settings.getInt(IntGlobalSetting.lastVisitedAccountId);
+    if (id == null) return null; // No account has been visited yet.
+
+    // (Will be null if `id` refers to an account that has been logged out.)
+    return _accounts[id];
```

Comment:
nit:
```suggestion
    return getAccount(id);
```

That way `_accounts` remains a bit more encapsulated among the methods that provide the list of accounts.